src/core/meshcop/secure_transport.cpp

```diff
@@ -606,14 +786,14 @@ int Dtls::HandleMbedtlsTransmit(const unsigned char *aBuf, size_t aLength)
     {
         LogDebg("HandleMbedtlsTransmit");
     }
-#if OPENTHREAD_CONFIG_COAP_SECURE_API_ENABLE
+#if OPENTHREAD_CONFIG_TLS_API_ENABLE
     else
     {
-        LogDebg("ApplicationCoapSecure HandleMbedtlsTransmit");
+        LogDebg("Application CoapSecure or Ble HandleMbedtlsTransmit");
```

Comment:
It should not say 'CoapSecure' here because the DTLS case is handled by the first if() block.
It also should not say 'Ble' because the secure_transport code here is supposed to be generic, not tied to BLE. 

So the first if() block implies DTLS is used, while the second else{} block implies that TLS is being used. So maybe have these respective messages:

```
LogDebg("HandleMbedtlsTransmit DTLS");
...
LogDebg("HandleMbedtlsTransmit TLS");
```

Reply:
fixed.